deploy/monitoring/observability-operator-rules.yaml

```diff
@@ -1,9 +1,5 @@
 ---
-# NOTE: For PrometheusRule ObO uses platform monitoring in OpenShift
-# instead of the forked PO's CRDs. As the ObO bundle ships the forked PO's CRD,
-# these won't be available at the time OLM installs the bundle
-
-apiVersion: monitoring.coreos.com/v1
```

Comment:
Not sure I follow...
The bundle contains the `monitoring.rhobs` CRDs. Shouldn't that suffice? Fwiw the e2e tests pass locally for me. Not sure why the tests fail in CI.

Reply:
[AFAIK, OLM installs can't refer to operator's own crds (see failure in CI). ](https://github.com/rhobs/observability-operator/actions/runs/4446674080/jobs/7807274076?pr=253#step:4:555) . I.E if an operator add a CRD Foobar, then its csv can't create resource `xyz` of kind `Foobar` because when the operator installs the bundle, it tries to create `xyz` and fails due to missing `Foobar` crd. 

The reason this passes on your machine could be because you already have these CRDs added. 

We should see if there is a workaround for this. This was also one of the historic reason why we relied on incluster monitoring for ObO. If ObO fails, then nothing will give us alerts.